sale_view_adjust_cft/data/ir.model.access.csv

```diff
@@ -0,0 +1,3 @@
+id,active,name,model_id/id,group_id/id,perm_read,perm_write,perm_unlink,perm_create
+__export__.ir_model_access_614,True,res.partner customerrank,sale_view_adjust_cft.model_res_partner_rank,base.group_user,True,False,False,False

```

Comment:
Please correct the ids.  The way you assign ids could potentially cause clash with existing records.

Ref: https://www.odoo.com/forum/help-1/question/what-are-these-in-ir-model-access-csv-12430#answer_12435